### PR TITLE
Fail-fast on timeout constraint violations during KafkaConsumer creation

### PR DESCRIPTION
**Pull request by harelba**
Related to #739

### CODE REVIEW

**Comment by harelba** [comment]
saw that there are failures in travis. I'm taking a look at it.

If you could provide instructions on how to run it locally with tox, that would be great.

**Comment by harelba** [comment]
fixed - Another build has started. Monitoring.

**Comment by jeffwidman** [comment]
Can we drop support for python 2.6? I'd rather be able to use `.format()`

**Comment by Drizzt1991** [comment]
I like the idea of such errors.

File: kafka/consumer/group.py [hunk 1]
```diff
@@ -267,6 +269,18 @@ def __init__(self, *topics, **configs):
                         new_config, self.config['auto_offset_reset'])
             self.config['auto_offset_reset'] = new_config
 
+        request_timeout_ms = self.config['request_timeout_ms']
+        session_timeout_ms = self.config['session_timeout_ms']
+        fetch_max_wait_ms = self.config['fetch_max_wait_ms']
+        if request_timeout_ms <= session_timeout_ms:
+            raise KafkaConfigurationError(
+                "Request timeout (" + str(request_timeout_ms) + ") must be larger than session timeout (" +
```
Comment (dpkp) [inline kafka/consumer/group.py]: `"Request timeout (%s) must be larger than session timeout (%s)" % (request_timeout_ms, session_timeout_ms)` is another alternative here

Comment (harelba) [inline kafka/consumer/group.py]: true, and better of course. fixing.

**Comment by harelba** [comment]
fixed string interpolation according to comment. Travis running.

File: kafka/consumer/group.py (hunk 1 above)
Comment (harelba) [inline kafka/consumer/group.py]: done, pushed fix.

**Comment by dpkp** [comment]
Thanks!

**Pull request merged**